Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,46 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    mkdir "build" do
```

Comment:
did you try an in-tree build?

Reply:
I’m not quite sure what you mean. I basically just followed the upstream instruction. Or so I think. Care to elaborate?

Reply:
meaning
```
    system "cmake", ".", *std_cmake_args
    system "make", "install"
```
and no `mkdir "build" do`.

Reply:
Yay, your suggestion works just as well! Fixed.